packages/app/src/debug/DebugSpec.vue

```diff
@@ -16,11 +16,64 @@
           class="flex w-full grid px-18px gap-y-8px items-center"
         >
           <div class="flex-grow flex w-full gap-x-2 truncate items-center">
-            <IconDocumentText
-              stroke-color="gray-500"
-              fill-color="gray-100"
-              size="16"
-            />
+            <Tooltip
+              v-if="foundLocally"
+              placement="bottom"
+              color="dark"
+              data-cy="open-in-ide"
+            >
+              <OpenFileInIDE
+                v-slot="{onClick}"
+                :file-path="specData.fullPath"
+              >
```

Comment:
When the user opens the IDE and then returns to the app, the tooltip reshows itself. Can this be closed on click or I saw there is an `auto-hide` prop for the tooltip component we are using.

https://user-images.githubusercontent.com/1702361/216642931-92e34ffa-d5d0-4399-a3f8-dd0fb2847591.mov

Reply:
I would say this is expected behavior. The tooltip shows on hover/focus, when you click back into the app the browser restores focus to the button, causing the tooltip to appear. You could probably workaround it, but I don't think the workaround would be worth the investment